src/Symfony/Component/Form/Tests/Extension/Validator/Constraints/FormValidatorTest.php

```diff
@@ -123,6 +123,19 @@ public function testDontValidateIfParentWithoutCascadeValidation()
         $this->assertNoViolation();
     }
 
+    public function testNotExistingConstraintIndex()
+    {
+        $object = $this->getMock('\stdClass');
+        $options = array();
```

Comment:
You can omit creating this variable as `FormBuilder`'s last parameter is `array $options = array()`